components/datetime/src/combo.rs

```diff
@@ -66,232 +72,67 @@ use icu_provider::marker::NeverMarker;
 ///     "Fri, 3:44 PM Los Angeles Time"
 /// );
 /// ```
-#[derive(Debug)]
-pub struct Combo<D, T, Z> {
-    _d: PhantomData<D>,
-    _t: PhantomData<T>,
+///
+/// Mix a dynamic [`DateFieldSet`](crate::fieldset::dynamic::DateFieldSet)
+/// with a static time zone:
+///
+/// ```
+/// use icu::datetime::fieldset::{YMD, dynamic::DateFieldSet};
+/// use icu::datetime::DateTimeFormatter;
+/// use icu::locale::locale;
+/// use icu::timezone::IxdtfParser;
+/// use writeable::assert_try_writeable_eq;
+///
+/// let formatter = DateTimeFormatter::try_new(
+///     &locale!("en-US").into(),
+///     DateFieldSet::YMD(YMD::long()).v(),
```

Comment:
nit: comment saying that this is Combo<YMD, V>

Reply:
Will address this in #5807